Magento_Paypal/templates/express/review.phtml

```diff
@@ -0,0 +1,263 @@
+                                    "
+                                >
+                                    <?= $block->escapeHtml(__('Edit')) ?>
+                                </a>
+                            </div>
+                        </div>
+                    <?php endif; ?>
+                </div>
+            <?php endif; ?>
+            <div
+                class="
+                    payapal-review__details-item
+                    col-lg-3
+                    margin-bottom-m
+                "
+            >
+                <h2 class="heading--third-level">
+                    <?= $block->escapeHtml(__('Payment Method')) ?>
+                </h2>
+                <div class="box-content">
+                    <?= $block->escapeHtml($block->getPaymentMethodTitle()) ?>
+                    <br>
+                    <?= $block->escapeHtml($block->getEmail()) ?>
+                </div>
+                <?php if ($block->getEditUrl()) : ?>
+                    <div class="actions-group">
+                        <div class="actions-group__handler">
+                            <a
+                                href="<?= $block->escapeUrl($block->getEditUrl()) ?>"
+                                class="
+                                    actions-group__link
+                                    padding-top-xs
+                                    action
+                                    edit
+                                "
+                            >
+                                <?= $block->escapeHtml(__('Edit Payment Information')) ?>
+                            </a>
+                        </div>
+                    </div>
+                <?php endif ?>
+            </div>
+        </div>
+    </div>
+
+    <div
+        class="
+            paypal-review__discount
+            row
+            margin-bottom-xl
+        "
+    >
+        <div class="col-lg-5">
+            <?= $block->getChildHtml('paypal.additional.actions') ?>
+        </div>
+    </div>
+
+    <div class="payapal-review__shopping-cart">
+        <h2 class="heading--third-level">
+            <?= $block->escapeHtml(__('Items in your shopping cart')) ?>
+        </h2>
+        <div class="actions-group">
+            <div class="actions-group__handler">
+                <a
+                    href="<?= $block->escapeUrl($block->getEditUrl()) ?>"
+                    class="actions-group__link padding-top-0 action edit"
+                >
+                    <?= $block->escapeHtml(__('Edit Shopping Cart')) ?>
+                </a>
+            </div>
+        </div>
+        <?= $block->getChildHtml('details') ?>
+
+        <form
+            method="post"
+            id="order-review-form"
+            action="<?= $block->escapeUrl($block->getPlaceOrderUrl()) ?>"
+            class="form order-review-form"
+        >
+            <?= $block->getChildHtml('agreements') ?>
+            <button
```

Comment:
please add original div with `<div class="actions-toolbar" id="review-buttons-container">`
I think You can create `actions-group` component same as above.
Also check the `Place Order` button. I think it should not be 100% widh on desktop view:

Reply:
I'm not sure about changing button width because it looks like it's full width on desktop in this solution https://gyazo.com/6a920a16790117df08f65cfc9671fec5 and this is supposed to be similar. Putting it inside `actions-group` component will cause the button width to shrink.

Reply:
ok, let's keep it like that